### DISCUSSION TITLE
How to run with windows

### DISCUSSION THREAD

**Discussion by acidiic** [rdeepak2002/reddit-place-script-2022#275] [Q&A] [answered]
sorry I am not experience with coding, when i try and start it the cmd opens and closes instantly and nothing runs. Am i having issues with starting it or is it something else that's not working?

**Comment by rdeepak2002** [comment, marked as answer]
Hello, please make sure you have Python installed on your machine: https://www.python.org/downloads/

Then please run the following commands and paste the output here:

```python
pip install -r requirements.txt
python main.py --debug
```

Also ensure that your config.json file is correct and make sure the JSON formatting is correct by pasting the contents of the config.json file into this tool: https://jsonlint.com/ and clicking validate
